backend/src/v4/models/notification.js

```diff
@@ -142,7 +142,7 @@ const fillModelData = async function(fullNotifications) {
 	notifications.forEach (notification => {
 		const teamSpace = modelsData[notification.teamSpace] || {};
 		const {name, federate} = teamSpace[notification.modelId] || {};
-		Object.assign(notification, {modelName: name, federation: federate, project: (modelToProject[notification.teamSpace] ?? {})[notification.modelId]});
+		Object.assign(notification, {timestamp: new Date(notification.timestamp).getTime(), modelName: name, federation: federate, project: (modelToProject[notification.teamSpace] ?? {})[notification.modelId]});
```

Comment:
I think you can just do this? Unless i'm missing something there shouldn't be a situation where the timestamp is not a Date object?

```suggestion
		Object.assign(notification, {timestamp: notification.timestamp.getTime(), modelName: name, federation: federate, project: (modelToProject[notification.teamSpace] ?? {})[notification.modelId]});
```